AsyncDisplayKit/Details/ASBatchContext.h

```diff
@@ -34,6 +34,14 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)completeBatchFetching:(BOOL)didComplete;
 
 /**
+ * Let the context object know that a batch fetch was completed.
+ *
+ * @discussion For instance, when a table has reached the end of its data, a batch fetch will be attempted unless the context
+ * object thinks that it is still fetching.
+ */
+- (void)completeBatchFetching;
```

Comment:
See the discussion in the initial issue #341 to deprecate the old method where we pass in a BOOL value. I don't see a value for the old method to be honest as if you pass in NO nothing will basically happen. I didn't mark the old method as deprecated yet in the header file though, but we should think about that for 2.0